app/navigation/NotificationNavigator.scala

```diff
@@ -36,7 +37,12 @@ class NotificationNavigator @Inject()() {
     case LetterReferencePage => _ => routes.RelatesToController.onPageLoad(NormalMode)
 
     case RelatesToPage => _ => routes.AreYouTheIndividualController.onPageLoad(NormalMode)
-  
+
+    case AreYouTheIndividualPage => ua => ua.get(AreYouTheIndividualPage) match {
+      case Some(_) => routes.OffshoreLiabilitiesController.onPageLoad(NormalMode)
+      case None => routes.AreYouTheIndividualController.onPageLoad(NormalMode)
+    }
+
```

Comment:
So here we don't need to cater for there not being any data - the rest of our functionality should provide for that.

Here we can follow the pattern above of:
`case AreYouTheIndividualPage => _ => routes.OffshoreLiabilitiesController.onPageLoad(NormalMode)`